unsymlink-lib

```diff
@@ -164,7 +164,11 @@ class MigrationState(object):
         self.eroot = eroot
 
     def analyze(self, usr_merge, real_prefixes):
-        from portage import create_trees, _encodings
+        from portage import create_trees, const, _encodings
+
+        eprefix = const.EPREFIX.encode(_encodings['fs'])
+        assert self.eroot.endswith(eprefix), "EROOT must be ROOT/EPREFIX."
+        root = self.eroot[:-len(eprefix)]
```

Comment:
Not sure if we don't have to account for extra slashes, esp. using `--root /foo/bar/`.

Reply:
Let's do an `os.path.normpath()` first.